Trading/Mode/signal_trading_mode/config/SignalTradingMode.json

```diff
@@ -3,6 +3,7 @@
     "required_strategies": [
         "MoveSignalsStrategyEvaluator"
     ],
+    "max_currency_percent": 33,
```

Comment:
If a user uses only 1 symbol, SignalTradingMode will never trade more than 33% of its portfolio right ?

Reply:
with default values, right

Reply:
maybe we should put it a 100 by default actually, so that it can use all of it

Reply:
I think it's better, then if people want to change it, they can